### ISSUE TITLE
Check if landcovers are mapped appropriately when post-processing HEET inputs

### ISSUE THREAD

**Issue by tomjanus** [tomjanus/reemission#29]
Looged by @cdb0101 

"I believe the heet output field names have changed for some parameters with different heet versions, namely landcover categories. Entirely hypothetical example : r_landcover_bysoil_2 : in earlier heet versions pertained to 'crop'; in most recent heet version pertains to 'grass'. This may be the reason why the reemission computed pre-impoundment CO2 fluxes appear to be 0 for all reservoirs; i.e. incorrect landcover classes applied to respective coeffs.
It would be best if we could verify that the heet field names for landcover classes do actually contain the data that they are supposed to (i.e. against  the output specification but this would mean digging into the code; https://urldefense.com/v3/__https://github.com/UoMResearchIT/heet/blob/main/docs/99_output_specification.csv__;!!PDiH4ENfjr2_Jw!CtxuUhFW08hyTJSbLMe8IIdxDTQCP5xV-_JHjRR_rEmk-benRmMvRykBAM7Z6NmvMg1d-_CBZiiuoxfsoVpqyOeyJPI$ [github[.]com]) - As far as I can tell they look correct though.
In summary -  check if reemission needs recoding via the output specification csv. When going through my initial R script of calcs I had to make such changes so I expect you may need to too."

**Comment by tomjanus** [comment]
Can we verify that this is indeed the case? Re-emission code has been updated to reflect this change but I am still not convinced if the landuse order needs to be swapped. Zero pre-impoundment fluxes were due to a different bug unrelated to this.

**Comment by cdb0101** [comment]
The present version of heet and associated documentation has correct output specifications; i.e. the heet output csv fields correctly match those of the output specification file fields.
Some of these codes did change with older versions of heet, so I just wanted to check that REEMISSION was updated to use the correct heetoutput fields in calculations, as this was a potential reason for discrepancies in pre-impoundment emissions. 

[please note from the initial comment this was an example with relation to reality, " **Entirely hypothetical example** : r_landcover_bysoil_2 : in earlier heet versions pertained to 'crop'; in most recent heet version pertains to 'grass'".

**Comment by tomjanus** [comment]
@cdb0101 - thanks for the clarification. From what I understand, the reemission code might have been correct and the swap of landcover indices in one of the recent commits should probably be reverted. It's possibly the cause of discrepancy between reemission results and the R script. I'm reopening this issue and shall close after more investigation.

**Comment by tomjanus** [comment]
Addressed in [581a3c4](https://github.com/tomjanus/reemission/commit/581a3c47c95409af96b6a448a81ac158dcdb6c0f)
